packages/amplify-category-function/src/provider-utils/awscloudformation/utils/cloudformationHelpers.ts

```diff
@@ -1,9 +1,9 @@
 import { category as categoryName } from '../../../constants';
 
-export function getNewCFNEnvVariables(oldCFNEnvVariables, currentDefaults, newCFNEnvVariables, newDefaults) {
+export function getNewCFNEnvVariables(oldCFNEnvVariables, currentDefaults, newCFNEnvVariables, newDefaults, amplifyMeta) {
```

Comment:
can use StateManager.getCurrentMeta() from amplify-cli-core instead of getting it from the context